src/Message/Headers.php

```diff
@@ -48,7 +48,7 @@ private function parseHeader($key, $value)
             case 'draft':
                 // no break
             case 'unseen':
-                return (bool)trim($value);
+                return $value;
```

Comment:
Flags can't be force to boolean because they can have multiple values, like `recent`:

https://secure.php.net/manual/en/function.imap-headerinfo.php